integrated_channels/sap_success_factors/utils.py

```diff
@@ -0,0 +1,96 @@
+"""
+Utilities for Enterprise Integrated Channel SAP SuccessFactors.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import datetime
+import json
+
+from django.apps import apps
+
+from enterprise.lms_api import parse_lms_api_datetime
+from integrated_channels.integrated_channel.course_metadata import BaseCourseExporter
+
+
+UNIX_EPOCH = datetime.datetime(1970, 1, 1)
+UNIX_MIN_DATE_STRING = '1970-01-01T00:00:00Z'
+UNIX_MAX_DATE_STRING = '2038-01-19T03:14:07Z'
+
+
+class SapCourseExporter(BaseCourseExporter):
+    """
+    Class to provide data transforms for SAP SuccessFactors course export task.
+    """
+
+    def __init__(self, user, plugin_configuration):
+        super(SapCourseExporter, self).__init__(user, plugin_configuration)
+
+    def get_serialized_data(self):
+        final_structure = {
+            'ocnCourses': self.courses
+        }
+        return json.dumps(final_structure, sort_keys=True).encode('utf-8')
```

Comment:
I don't know... I will ask SAP. In terms of where that will live, I am leaning towards in the client. But this can stay for now and I can adjust when I stitch together these scripts with the transmitters/client.